### DISCUSSION TITLE
repo for rpm package

### DISCUSSION THREAD

**Discussion by talsh5528** [Mirantis/cri-dockerd#287] [General]
there is any way to create a repo or something like that to manage the rpm's packages? instead of download the packages manually from https://github.com/Mirantis/cri-dockerd/releases and then Assets ?
if I want to download as part of my ci the newest version of cri-dockerd rpm , how I can do that?

**Comment by afbjorklund** [comment]
* #131

As far as I know, you currently have to "know" the package version.

**Reply by talsh5528** [reply]
this is the version: cri-dockerd-0.3.8-3.el8.x86_64.rpm
I will be able to find this package's newest version in the repo if I will configure this repo?
`cat <<EOF | tee /etc/yum.repos.d/cri-o.repo
[cri-o]
name=CRI-O
baseurl=https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main/rpm/
enabled=1
gpgcheck=1
gpgkey=https://pkgs.k8s.io/addons:/cri-o:/prerelease:/main/rpm/repodata/repomd.xml.key
EOF`

@afbjorklund

**Reply by afbjorklund** [reply]
No, not in that CRI-O repo.

Only from GitHub Releases

**Reply by talsh5528** [reply]
can I replace cri-dockerd.x86_64 in cri-o? 
because if I can install cri-dockerd.x86_64 only from GitHub Releases I don't have the option to add that to my CI process and update the package when I need ...
@afbjorklund

**Reply by afbjorklund** [reply]
That feature would be nice.

Thus, the issue above. #131

**Reply by talsh5528** [reply]
so we will wait for the feature to be created, did you have any experience using cri-o as your Kubernetes engine?
